src/NodaTime.Web.Test/Helpers/VersionComparerTest.cs

```diff
@@ -0,0 +1,23 @@
+﻿using NodaTime.Web.Helpers;
+using NUnit.Framework;
+
+namespace NodaTime.Web.Test.Helpers
+{
+    public class VersionComparerTest
+    {
+        [Test]
+        [TestCase("1.0.0", "1.0.1")]
+        [TestCase("1.0.0", "1.1.0")]
+        [TestCase("1.1.0", "2.0.0")]
+        [TestCase("1.0.0", "1.0.1-alpha01")]
+        [TestCase("1.0.0-alpha01", "1.0.0-alpha02")]
+        [TestCase("1.0.0-alpha01", "1.0.0-beta01")]
+        [TestCase("1.0.0-alpha01", "1.0.0")]
+        public void Compare(string earlier, string later)
```

Comment:
Just for completeness, add some pairs that should be equivalent? (which I think means that the strings need to be equal?)

Also perhaps include something with a two-digit major/minor/patch number? (even if it's to document that that comparison doesn't work properly).